glass/fields.py

```diff
@@ -35,36 +34,22 @@
 import numpy.typing as npt
 from gaussiancl import gaussiancl
 
-# types
-Size = typing.Optional[typing.Union[int, tuple[int, ...]]]
-Iternorm = tuple[typing.Optional[int], npt.NDArray[typing.Any], npt.NDArray[typing.Any]]
-ClTransform = typing.Union[
-    str, typing.Callable[[npt.NDArray[typing.Any]], npt.NDArray[typing.Any]]
-]
-Cls = collections.abc.Sequence[
-    typing.Union[npt.NDArray[typing.Any], collections.abc.Sequence[float]]
-]
-Alms = npt.NDArray[typing.Any]
+if typing.TYPE_CHECKING:
+    import collections.abc
 
 
 def iternorm(
     k: int,
-    cov: collections.abc.Iterable[npt.NDArray[typing.Any]],
-    size: Size = None,
-) -> collections.abc.Generator[Iternorm, None, None]:
+    cov: collections.abc.Iterable[npt.NDArray[np.float64]],
+    size: tuple[int, ...] = (),
```

Comment:
this was correct before: `size` has type `int | tuple[int, ...] | None` and default should remain `None`, which is semantically different from `()`

Reply:
As discussed, I'm deprecating the `None` option. But have restored the type as `int | tuple[int, ...]`